servicebus/spring-cloud-azure-stream-binder-servicebus/servicebus-queue-binder-arm/terraform/main.tf

```diff
@@ -0,0 +1,104 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "2.99.0"
+    }
+    azurecaf = {
+      source  = "aztfmod/azurecaf"
+      version = "1.2.10"
+    }
+    azuread = {
+      source  = "hashicorp/azuread"
+      version = "2.19.0"
+    }
+    random = {
+      source  = "hashicorp/random"
+      version = "3.1.0"
+    }
+  }
+}
+
+data "azuread_client_config" "current" {}
+
+# Configure the Azure Active Directory Provider
+provider "azuread" {
+}
+
+# Configure an app
+resource "azuread_application" "servicebusqueuebinder" {
+  display_name = "servicebus-queue-binder-arm"
```

Comment:
add a random string here for the display_name

like display_name = "servicebus-queue-binder-arm-xxxxx"

Reply:
you can refer to https://registry.terraform.io/providers/hashicorp/random/latest/docs